code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -204,4 +204,169 @@ datum/reagent/itching_powder/on_mob_life(var/mob/living/M as mob)
 	result = "facid"
 	required_reagents = list("sacid" = 1, "fluorine" = 1, "hydrogen" = 1, "potassium" = 1)
 	result_amount = 4
-	required_temp = 380
\ No newline at end of file
+	required_temp = 380
+
+datum/reagent/regadenoson
+	name = "Regadenoson"
+	id = "regadenoson"
+	description = "Major stamina regeneration buff. 33% chance to hit with 5-25 TOX. 5-10% chances to stun and cause suffocation or immediate heart failure."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.4
+
+datum/reagent/regadenoson/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustStaminaLoss(-5)
+	if(prob(33))
+		M.adjustToxLoss(rand(5,25))
+	if(prob(rand(5,10)))
+		var/picked_option = rand(1,3)
+		switch(picked_option)
+			if(1)
+				M.Stun(3)
+				M.Weaken(3)
+			if(2)
+				M.losebreath += 10
+				M.adjustOxyLoss(rand(5,25))
+			if(3)
+				M.visible_message("<span class = 'userdanger'>[M] clutches at their chest as if their heart stopped!</span>")
+				M.adjustBruteLoss(100) // rip in pepperoni
+	..()
+	return
+
+datum/reagent/pancuronium
+	name = "Pancuronium"
+	id = "pancuronium"
+	description = "10 cycles to paralysis, 7% chance to cause 3-5 LOSEBREATH."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.2
+	var/current_cycle = 0
+
+datum/reagent/pancuronium/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	current_cycle++
+	if(current_cycle == 10)
+		M.SetParalysis(10)
+	if(prob(7))
+		M.losebreath += rand(3,5)
+	..()
+	return
+
+datum/reagent/sodium_thiopental
+	name = "Sodium Thiopental"
+	id = "sodium_thiopental"
+	description = "KOs in ten cycles. Major total stamina penalty."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.7
+	var/current_cycle = 0
+
+datum/reagent/sodium_thiopental/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	current_cycle++
+	if(current_cycle == 10)
+		M.sleeping += 20
+	M.adjustStaminaLoss(10)
```

Comment:
we have a lot of reagents that drain stamina now actually, and disablers are a lot more common